modules/OFConnectionManager2/module/src/cxn_instance.h

```diff
@@ -228,6 +228,17 @@ typedef struct connection_s {
         uint32_t xid;   /* XID of barrier request */
     } barrier;
 
+    /* async opeartions */
+    uint32_t async_pending_cnt;
+    uint32_t iter_pending_cnt;
+
+    /* Bundle Async op */
+    // bool bundle_async_pendingf;  /* bundle async op pending flag */
+
+    /* iter Async op (gentable only) */
+    bool iter_async_pendingf;  /* iteration task async op pending flag */
+    // bool iter_async_donef;     /* iteration task async op done flag */
```

Comment:
i think i missed something obvious -- how many pending operations will we support?  do we need to be able to handle more than one?  or is one good enough (although it could potentially be very slow)?
also, i am confused as to why there are three variables here -- two counts and a flag.  i was thinking that we only needed to block the bundle task from continuing.  if there is only one pending operation, then we only need one flag.  if we want to support more than one pending operation in the same bundle, then we don't need a count instead of a flag.
my suggestion is to reduce the scope of the change as much as possible so that it is as simple as possible to understand and debug.